back/src/Controller/IoSocketController.ts

```diff
@@ -72,7 +71,20 @@ export class IoSocketController {
                 clientEventsEmitter.registerToClientLeave(ws.clientLeaveCallback);
             },
             message: (ws, arrayBuffer, isBinary): void => {
-                console.log('m', ws); //todo: add admin actions such as ban here
+                try {
+                    //TODO refactor message type and data
+                    const message: {event: string, message: {type: string, message: unknown, userUuid: string}} =
+                        JSON.parse(new TextDecoder("utf-8").decode(new Uint8Array(arrayBuffer)));
+
+                    if(message.event === 'user-message') {
+                        if (message.message.type === 'ban') {
```

Comment:
Since this feature is used to "ban" a user, should we not at some point in the code have a "ws.close()" call to forcefully close the connection from the server side?